interface/patient_file/summary/demographics.php

```diff
@@ -1787,4 +1770,27 @@ function setMyPatient() {
   //show failure message.
 }
 
+//image mime check with all image formats.
+function has_right_mime($image_properties) {
+  $mime = $image_properties["mime"];
```

Comment:
@prondubuisi nice idea to me. Just a thought we have many places where uploads is being done. Why not write this as a library then import so that we use them everywhere in the codebase. Just a suggestion. Lets here what @aethelwulffe has to say